RadarSDK/Include/Radar.h

```diff
@@ -522,7 +522,8 @@ completionHandler:(RadarSendEventCompletionHandler)completionHandler NS_SWIFT_NA
 
  @see https://radar.com/documentation/trip-tracking
  */
-+ (void)startTripWithOptions:(RadarTripOptions *_Nonnull)options NS_SWIFT_NAME(startTrip(options:));
++ (void)startTripWithOptions:(RadarTripOptions *_Nonnull)options
+    NS_SWIFT_NAME(startTrip(options:)) __deprecated_msg("Use +[Radar startTripWithOptions:trackingOptions:completionHandler:] instead.");
```

Comment:
@corypisano @jrtibbetts Are we deprecating this on Android as well? Do we need to?

Reply:
I'll un-deprecate it.